semgrep/tests/e2e/snapshots/test_check/test_max_memory/error.txt

```diff
@@ -36,10 +36,10 @@ Files skipped:
    • <none>
 
 
-skipped: all .gitignored files
-for a detailed list of skipped files, run semgrep with the --verbose flag
+Some files were skipped.
+  Scan was limited to files tracked by git.
```

Comment:
To me this is notably worse than the previous message, which explained how I could affect the outcome (edit the gitignore). My suggestion:

```
Skipped some files:
* skipped files not tracked by git (using .gitignore)
```

Reply:
My original issue here was that I couldn't find a simple way to word

> Skipped files that are unchanged since your baseline commit.

"skipped" and "unchanged" is kind of a double negative which never doesn't hurt understanding.

Then I just made the `git ls-files` message the same as the `git diff` message.

Reply:
so actually I disagree with

> worse than the previous message, which explained how I could affect the outcome (edit the gitignore)

in fact I think it gave you false confidence, because

1. you cannot skip a file by just .gitignore-ing it, you also need to untrack it
2. you cannot unskip a file by removing the .gitignore entry, you also need to track it